src/Components/Users/ManageUsers.tsx

```diff
@@ -1,4 +1,4 @@
-import { useCallback, useState } from "react";
+import React, { useCallback, useState } from "react";
```

Comment:
Importing `React` is not required here, we're already importing the necessary functions.

```suggestion
import { useCallback, useState } from "react";
```

Tested and working fine.

Reply:
I have added `React` to fix Deepscan issue